Sources/Commands/SwiftTestCommand.swift

```diff
@@ -124,6 +124,11 @@ struct TestEventStreamOptions: ParsableArguments {
     @Option(name: .customLong("experimental-attachments-path"),
             help: .private)
     var experimentalAttachmentsPath: AbsolutePath?
+
+    /// Path for writing attachments (Swift Testing only.)
+    @Option(name: .customLong("attachments-path"),
+            help: "Path where attachments should be written (Swift Testing only)")
```

Comment:
I'd add a note about the default attachment path when this is omitted, indicating this flag isn't required to use attachments. You only need it if you want them to go somewhere custom.

Reply:
There is no default path. If you don't specify it, we don't save attachments.

Reply:
_That_ might still be worth noting in the description.

Reply:
e.g. "If unspecified, attachments will not be written to disk."

Reply:
Fair. Gonna let CI finish before modifying the string just because it takes so long.